dove/tests/helper.rs

```diff
@@ -0,0 +1,181 @@
+#![allow(dead_code)]
```

Comment:
Should it be removed? If there's code that is unused and covered with this attribute, should be removed too.